File: Examples/Node.js/SDK/Worksheet/GetAutoshape.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var StorageApi = require('asposestoragecloud');
var CellsApi = require('asposecellscloud');
var configProps = require('../Data/config.json');
var data_path = '../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose Cells API SDK
var cellsApi = new CellsApi(config);

// Set input file name
var name = "Sample_Test_Book.xls";
var sheetName = "Sheet4";
var autoshapeNumber = 1;

try {
// Upload source file to aspose cloud storage
storageApi.PutCreate(name, null, null, data_path + name , function(responseMessage) {
	assert.equal(responseMessage.status, 'OK');
	
	// Invoke Aspose.Cells Cloud SDK API to get autoshape from a worksheet
	cellsApi.GetWorksheetAutoshape(name, sheetName, autoshapeNumber,  null, null, function(responseMessage) {
			assert.equal(responseMessage.status, 'OK');
			var autoShape = responseMessage.body.AutoShape;
			console.log(autoShape.HtmlText);
			});
	});

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1

File: Examples/Android/app/src/main/java/com/aspose/cells/cloud/examples/worksheet/GetAutoshape.java
package com.aspose.cells.cloud.examples.worksheet;


import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;
import java.io.IOException;
import java.io.File;
import com.aspose.cells.cloud.examples.R;
import android.content.Context;


public class GetAutoshape {

	public static void execute(Context context) throws IOException {
		//ExStart: get-auto-shape
		try {
			// Instantiate Aspose Storage API SDK
			StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

			// Instantiate Aspose Words API SDK
			CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
			String input = "Sample1.xlsx";
			File inputFile = Utils.stream2file("sample1","xlsx", context.getResources().openRawResource(R.raw.sample1));
			String sheet = "Sheet1";
			int autoshape = 0;

			storageApi.PutCreate(input, null, Utils.STORAGE, inputFile);

			com.aspose.cells.model.AutoShapeResponse ar = cellsApi
					.GetWorksheetAutoshape(input, sheet, autoshape, Utils.STORAGE, null);

			System.out.println("Autoshape HTML: " + ar.getAutoShape().getHtmlText());
		}

		catch (Exception e) {
			e.printStackTrace();
		}
		//ExEnd: get-auto-shape
	}
}